specs/MultiProfile.md

```diff
@@ -426,9 +472,9 @@ namespace Microsoft.Web.WebView2.Core
 
         CoreWebView2CookieManager CookieManager { get; };
         
-        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile3")]
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2StagingProfile4")]
```

Comment:
This should use the expected public name eg ICoreWebView2Profile4 (or whatever the next ProfileN number is)

Reply:
OK, has remove all the `staging` keyword.